pspamm/codegen/architectures/arm/generator.py

```diff
@@ -69,8 +69,8 @@ def bcst_alpha_beta(self,
                         ) -> Block:
 
         asm = block("Broadcast alpha and beta so that efficient multiplication is possible")
-        asm.add(mov(alpha_reg[0], alpha_reg[1], True))
-        asm.add(mov(beta_reg[0], beta_reg[1], True))
+        # asm.add(mov(alpha_reg[0], alpha_reg[1], True))
```

Comment:
remove commented block?

Reply:
The instruction was incorrect, causing compile errors. I'll try to maybe find an alternative instruction here.

(again, pointing to the need of a CI)